esdt-system-sc-mock/src/esdt_system_sc_mock.rs

```diff
@@ -19,7 +19,7 @@ pub trait PayableFeatures {
         token_ticker: ManagedBuffer,
         initial_supply: BigUint,
         _num_decimals: usize,
-        #[var_args] _token_properties: MultiValueEncoded<MultiValue2<ManagedBuffer, bool>>,
+        _token_properties: MultiValueEncoded<MultiValue2<ManagedBuffer, bool>>,
```

Comment:
what's the naming convention here (some params with `_` other without)? it's a bit weird/random or am I missing something?

Reply:
Underscore in front means it's intentionally not used.

Reply:
This basically suppresses the "unused argument" warning. It's a Rust convention.